components/carousels/VerticalCarousel.vue

```diff
@@ -0,0 +1,58 @@
+<template>
```

Comment:
Please delete changes related carousels from this PR

Reply:
Done